taskcluster/docker/android-qt6-build/Dockerfile

```diff
@@ -68,9 +68,8 @@ ENV ANDROID_NDK_ROOT="/opt/NDK/${NDK_VERSION}"
 ENV ANDROID_NDK_HOME="/opt/NDK/${NDK_VERSION}"
 ENV QT_HOST_PATH=/opt/${QT_VERSION}/gcc_64
 
-ENV PATH="/opt/${QT_VERSION}/gcc_64/bin:${QTPATH}/bin:/opt/cmdline-tools/bin:/opt/android/sdk/cmake/${CMAKE_VERSION}/bin:${PATH}"
+ENV PATH="/opt/${QT_VERSION}/gcc_64/bin:${QTPATH}/bin:/opt/cmdline-tools/bin:/opt/android/sdk/cmake/${CMAKE_VERSION}/bin:${ANDROID_NDK_ROOT}/toolchains/llvm/prebuilt/linux-x86_64/bin:${PATH}"
```

Comment:
I really tried to do this automatically somehow, but then I gave up :/

Reply:
Totally fine, but I think maybe this should be reflected in the readme :)

Reply:
Agreed! I'll add that asa comment to https://mozilla-hub.atlassian.net/browse/VPN-3142, because for the README/md Android build content we have right now this won't make sense :sweat_smile: